source/Halibut.Tests/ServiceModel/PendingRequestQueueFixture.cs

```diff
@@ -379,47 +382,22 @@ public async Task QueueAndWait_CancellingAPendingRequestAfterItIsDequeued_Should
             // Cancel, and give the queue time to start waiting for a response
             cancellationTokenSource.Cancel();
             await Task.Delay(1000, CancellationToken);
+            dequeued!.CancellationToken.IsCancellationRequested.Should().BeTrue("Should have cancelled the request");
 
-            await sut.ApplyResponse(expectedResponse, request.Destination);
-
-            var response = await queueAndWaitTask;
-
+            
+#pragma warning disable VSTHRD003
```

Comment:
Like before, you can use the overload that just takes a task to remove the need for adding the pragma.